+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environment preparation.'
+ message='Start environment preparation.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start environment preparation.'
Tue Aug  8 10:59:38 2023 - INFO  - Start environment preparation.
+ OLD_LANG=
+ export LANG=en_US.UTF-8
+ LANG=en_US.UTF-8
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Tue Aug  8 10:59:40 2023 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Tue Aug  8 10:59:42 2023 - INFO  - Start to run test.
+ echo 'my name is root, directory is /root'
+ envsubst
+ cat
+ grep root
oe_test_envsubst.sh: line 31: envsubst: command not found
+ CHECK_RESULT 1 0 0 'Variable resolution failure'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Variable resolution failure'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Variable resolution failure'
+ LOG_ERROR 'Variable resolution failure'
+ message='Variable resolution failure'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Variable resolution failure'
Tue Aug  8 10:59:44 2023 - ERROR - Variable resolution failure
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_envsubst.sh line 32'
+ message='oe_test_envsubst.sh line 32'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_envsubst.sh line 32'
Tue Aug  8 10:59:45 2023 - ERROR - oe_test_envsubst.sh line 32
+ '[' 0 -eq 1 ']'
+ return 0
+ envsubst -h
oe_test_envsubst.sh: line 33: envsubst: command not found
+ grep envsubst
+ CHECK_RESULT 1 0 0 'check command fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='check command fail'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'check command fail'
+ LOG_ERROR 'check command fail'
+ message='check command fail'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'check command fail'
Tue Aug  8 10:59:47 2023 - ERROR - check command fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_envsubst.sh line 34'
+ message='oe_test_envsubst.sh line 34'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_envsubst.sh line 34'
Tue Aug  8 10:59:48 2023 - ERROR - oe_test_envsubst.sh line 34
+ '[' 0 -eq 1 ']'
+ return 0
+ envsubst -V
oe_test_envsubst.sh: line 35: envsubst: command not found
+ grep 'GNU gettext-runtime'
+ CHECK_RESULT 1 0 0 'check command fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='check command fail'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'check command fail'
+ LOG_ERROR 'check command fail'
+ message='check command fail'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'check command fail'
Tue Aug  8 10:59:50 2023 - ERROR - check command fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_envsubst.sh line 36'
+ message='oe_test_envsubst.sh line 36'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_envsubst.sh line 36'
Tue Aug  8 10:59:51 2023 - ERROR - oe_test_envsubst.sh line 36
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Tue Aug  8 10:59:53 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 3
+ test 3 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Tue Aug  8 10:59:54 2023 - ERROR - failed to execute the case.
+ exit 3
+ post_test
+ LOG_INFO 'Start to clean the test environment.'
+ message='Start to clean the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to clean the test environment.'
Tue Aug  8 10:59:56 2023 - INFO  - Start to clean the test environment.
+ export LANG=
+ LANG=
+ LOG_INFO 'End to clean the test environment.'
+ message='End to clean the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to clean the test environment.'
Tue Aug  8 10:59:57 2023 - INFO  - End to clean the test environment.
